graspologic/preprocessing/graph_cuts.py

```diff
@@ -0,0 +1,383 @@
+# Copyright (c) Microsoft Corporation and contributors.
+# Licensed under the MIT License.
+
+import logging
+import random
+from typing import Any, Callable, List, NamedTuple, Optional, Tuple, Union
+import networkx as nx
+import numpy as np
+
+
+LARGER_THAN_INCLUSIVE = "LARGER_THAN_INCLUSIVE"
+LARGER_THAN_EXCLUSIVE = "LARGER_THAN_EXCLUSIVE"
+SMALLER_THAN_INCLUSIVE = "SMALLER_THAN_INCLUSIVE"
+SMALLER_THAN_EXCLUSIVE = "SMALLER_THAN_EXCLUSIVE"
```

Comment:
if the user is just passing strings at the end of the day, can they be lower case also?

Reply:
you pythonistas and your magic strings.  DO YOU HAVE NO LOVE FOR CODE COMPLETION?!?!?! 🤣